src/cmdstan/arguments/arg_output_sig_figs.hpp

```diff
@@ -12,15 +12,15 @@ class arg_output_sig_figs : public int_argument {
     _description
         = "The number of significant figures used for the output CSV files.";
     _validity
-        = "0 <= integer <= 18 or -1 to use the default number of significant "
-          "figures";
-    _default = "-1";
-    _default_value = -1;
+        = "0 <= integer <= 18, or -1 to use the system library's default "
+          "number of significant figures (usually 6).";
```

Comment:
shouldn't this `6` be an `8`?

Reply:
(Though I should note, if you want to use Stan’s default, a reasonable option is just not pass this argument at all…)

Reply:
I would prefer to just not allow -1 at all, but removing a valid value would definitely break at least somebody’s code

Reply:
Good point---I missed that was the old default!

Reply:
I've updated the wording. Here's what the full help text reads as now:

```
    sig_figs=<int>
      The number of significant figures used for the output CSV files.
      Valid values: 0 <= integer <= 18, or -1 to use the current operating system's default number of significant figures (This will vary, but on many systems it is 6).
      Defaults to 8
```